syncwatcher.go

```diff
@@ -422,26 +380,21 @@ func getFolders() []FolderConfiguration {
 	return cfg.Folders
 }
 
-// returns a function that takes a path as string and returns a boolean,
-// indicating whether the path should be ignored
-func ignoreTest(ignorePaths []string, ignorePatterns []Pattern,
-	folderPath string) func(string) bool {
-	return func(path string) bool {
-		relPath := relativePath(path, folderPath)
-		return shouldIgnore(ignorePaths, ignorePatterns, relPath)
-	}
-}
-
 // watchFolder installs inotify watcher for a folder, launches
 // goroutine which receives changed items. It never exits.
 func watchFolder(folder FolderConfiguration, stInput chan STEvent) {
 	folderPath := expandTilde(folder.Path)
-	ignorePatterns := getIgnorePatterns(folder.ID)
+	ignores := ignore.New(false)
+	Trace.Println("Getting ignore patterns for " + folder.ID)
+	ignores.Load(filepath.Join(folderPath, ".stignore"))
```

Comment:
This means we don't recursively load included files (e.g. `#include .stignoreglobal`) :/
Is the ignore rest API fully deprecated?